circle.yml

```diff
@@ -6,8 +6,37 @@ jobs:
       - image: quay.io/cbgeo/lbmdem
     steps:
       - checkout
-      # Configure CMake and run
+      # GCC 6.3
       - run:
-          name: Build LBM
-          command: mkdir build && cd build && cmake -DCMAKE_BUILD_TYPE=Release -DCMAKE_CXX_COMPILER=g++ -DCMAKE_EXPORT_COMPILE_COMMANDS=On .. && make clean && make -j8 && ./lbmdemtest
+          name: GCC-6 build
+          command: |
+            mkdir build && cd build
+            cmake -DCMAKE_BUILD_TYPE=Release -DCMAKE_CXX_COMPILER=g++ -DCMAKE_EXPORT_COMPILE_COMMANDS=On .. 
+            make clean && make -j8 
+            ./lbmdemtest
+      # GCC 3.9
+      - run:
+          name: Clang-3.9 build
+          command: |
+            cd build && rm -rf ./*
+            cmake -DCMAKE_BUILD_TYPE=Release -DCMAKE_CXX_COMPILER=clang++ -DCMAKE_EXPORT_COMPILE_COMMANDS=On .. 
+            make clean && make -j8 
+            ./lbmdemtest
```

Comment:
suggest `ctest -VV` rather than calling `lbmdemtest` directly for all the steps that run tests

would make adding more tests in the future a single global change